traffic_router/connector/src/main/java/com/comcast/cdn/traffic_control/traffic_router/protocol/RouterNioEndpoint.java

```diff
@@ -120,5 +121,13 @@ protected void doRun(){
 		}
 	}
 
+	public String getProtocols() {
+		return protocols;
+	}
+
+	public void setProtocols(final String protocols) {
+		this.protocols = protocols;
+	}
+
```

Comment:
I think this should be using the 'sslProtocol' attribute that is already part of the specification. Was there a problem with that?  I don't think its good to invent custom attributes.

Reply:
i used 'protocols' instead of 'sslProtocol' because thats how it was setup in TR version 2.2 and below.  its a valid Tomcat Connector attribute, but their docs admit it has some overlap with the 'sslProtocol'.

Reply:
I see you are correct that 'protocols' is a legitimate attribute. I think it would be nice to honor all of the attributes that are set in the `Connector` element while you are at it instead of selecting specific ones. That is why I've suggested this alternate implementation. We could circle back to this in the future.